driver/main_args.ml

```diff
@@ -339,6 +339,10 @@ let mk_nostdlib f =
   "-nostdlib", Arg.Unit f,
   " Do not add default directory to the list of include directories"
 
+let mk_nocwd f =
+  "-nocwd", Arg.Unit f,
+  " Do not add the current directory from the search path"
```

Comment:
```suggestion
  " Do not add the current directory from the load path"
```

Reply:
*to* the load path

Reply:
`from the load path` ⇒ `to the load path` ?

Reply:
I just added commit some small modification, I'll make another commit to change  "loading path" if necessary.